server/api/routes/ssot.js

```diff
@@ -29,12 +29,14 @@ router.get(
   '/getAllParameters/:robotId',
   variableController.retrieveParametersForRobot
 );
+router.delete('/deleteParameters', variableController.deleteMany);
```

Comment:
I'm not so happy about the naming deleteMany here, because it colides with th eMongoose query being used and it does not explain in detail what is deleted. Maybe `deleteParamtersForActivity`?

Reply:
Once the new API is integrated we will have to rename the route anyway, thats why I think it is okay for now :)